cpp/tests/groupby/group_count_scan_test.cpp

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <tests/groupby/groupby_test_util.hpp>
+
+#include <cudf_test/base_fixture.hpp>
+#include <cudf_test/column_wrapper.hpp>
+#include <cudf_test/type_lists.hpp>
+
+#include <cudf/detail/aggregation/aggregation.hpp>
+
+namespace cudf {
+namespace test {
+template <typename V>
+struct groupby_count_scan_test : public cudf::test::BaseFixture {
+};
+
+TYPED_TEST_CASE(groupby_count_scan_test, cudf::test::AllTypes);
+
+TYPED_TEST(groupby_count_scan_test, basic)
+{
+  using K = int32_t;
+  using V = TypeParam;
+  using R = cudf::detail::target_type_t<V, aggregation::COUNT_ALL>;
+
+  // clang-format off
+  fixed_width_column_wrapper<K> keys      { 1, 2, 3, 1, 2, 2, 1, 3, 3, 2};
```

Comment:
You can alias `fixed_width_column_wrapper<K>` and `fixed_width_column_wrapper<V>` (using the define macro) at the beginning of this file to shorten the rest of the file.

Reply:
Not a fan of macros.
Defined a macro. (Alternatively, type alias in typed fixture is also verbose).

Reply:
Please, no macros.